app/script/util/Environment.js

```diff
@@ -197,7 +204,7 @@ window.z.util = z.util || {};
         return app_version();
       }
 
-      const electron_version = this._electron_version(navigator.userAgent);
+      const electron_version = _check.get_electron_version(navigator.userAgent);
```

Comment:
This has nothing to do with my PR, instead it is related to this PR which you approved: https://github.com/wireapp/wire-webapp/pull/1455/files

But I'll fix it.